### ISSUE TITLE
update profile script: "are you sure"

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#377]
**Version:** r5_fix_222 d126e4c
**Environment:** tmaster

The update script has an "are you sure" step. This makes it more difficult to use in a scripted situation (and in general it is an annoyance to be asked to repeat yourself). 

```
This will update the current node using the from source code repository
using the profile: master
Do you want to continue? (y/n) [n]: n
```

**Comment by abhimukh** [comment]
Fixed in r5_fixes: cd43c397042deb5cefef03f2703bc20d014126cb ( The commit message has an incorrect issue number).  Added a timeout clause for batch environment.

**Comment by hansthen** [comment]
I am not so happy with these "are you sure" questions, from a usability perspective. The message adds no extra information than was present in the command, so why ask the user if they changed their mind in the mean time? They just delay the workflow.

**Comment by themisa** [comment]
Fixed: **Version:** r5_fixes c2dc5c0

```
[root@controller(amaster) trinity]# ./update controller
This will update the current node using the from source code repository
using the profile: controller
‘/root/trinity/controller/rootimg/cluster/.skel/.modulespath’ -> ‘/cluster/.skel/.modulespath’
```
